tedana/io.py

```diff
@@ -266,7 +266,9 @@ def save_tsv(self, data, name):
         data_type = type(data)
         if not isinstance(data, pd.DataFrame):
             raise TypeError(f"data must be pd.Data, not type {data_type}.")
-        data.to_csv(name, sep="\t", line_terminator="\n", na_rep="n/a", index=False)
+        if "rationale" in data.columns and "classification" in data.columns:
+            data.loc[data['classification'] == 'accepted', 'rationale'] = 'n/a'
```

Comment:
black will complain about the single quotes:
```suggestion
            data.loc[data["classification"] == "accepted", "rationale"] = "n/a"
```